### PR TITLE
Fixed course lessons so it's automated

### PR DESCRIPTION
**Pull request by lwasser**

### CODE REVIEW

**Comment by lwasser** [comment]
@mbjoseph as far as i can tell these links should not fail. they work built locally. one tho - the flood one is very slowly because it has interactive plots. i may replace those with images that link to the plots to help. please have a look.

**Comment by lwasser** [comment]
i have an idea... let's see if it's right! draft pages are getting in the way of my testing.

**Comment by lwasser** [comment]
ok @mbjoseph currently i'm having a dance party. Why? cause the build error is no longer on my darn learn page. But not that leaflet tutorial is coming up again. 

Full disclosure: i ran "make" i'm thinking that broke things maybe as it rebuild tutorials. please let me know how to handle this!!

**Comment by mbjoseph** [comment]
hmmmmmmmmm. Well I'm glad you're having a dance party. I will take a look tomorrow and investigate.

**Comment by mbjoseph** [comment]
Couldn't resist - build passing

**Comment by lwasser** [comment]
Hahahaa. Wow that's all kinda special dance parteee action right there.  I learned so much today !!

I know how to edit code highlighting now too!! Thanks. Can we merge this now to dev? If so I can then pr the breadcrumb code in one small piece!!

________________________________

File: _drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd [hunk 1]
```diff
@@ -7,7 +7,11 @@ authors: [Leah Wasser, NEON Data Skills]
 category: [course-materials]
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 class-lesson: ['intro-rmarkdown-knitr']
-permalink: /course-materials/intro-rmarkdown-code-chunks
+<<<<<<< HEAD:_drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd
```
Comment (mbjoseph) [inline _drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd]: Looks like this needs to be resolved

Comment (lwasser) [inline _drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd]: that's so weird. i've fixed that a few times. something is up with the way i'm resolving merge conflicts.

Comment (mbjoseph) [inline _drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd]: Yeah I'm curious to try out the new interactive editor on GitHub

Comment (lwasser) [inline _drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd]: interactive editor? how does that work?

Comment (lwasser) [inline _drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd]: it doesn't look as if i fixed that issue but i did. let me check elsewhere.

**Comment by mbjoseph** [comment]
Nice - I think this one's good to go.

**Comment by lwasser** [comment]
ok im going to merge. that is in drafts so when i rebuild it will cause problems if i have merge conflict code embedded. i'm going to merge! then i'll submit the breadcrumb PR. I promise much less to review.

**Comment by mbjoseph** [comment]
https://help.github.com/articles/resolving-a-merge-conflict-on-github/

**Comment by lwasser** [comment]
Cool – I’ll have to check that out! On a lesson building  mission today.

**Pull request merged**